server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -124,7 +117,7 @@ export class SummaryWriter implements ISummaryWriter {
         }
 
         // We should not accept a summary earlier than our current protocol state
-        if (op.referenceSequenceNumber < protocolSequenceNumber) {
+        if (op.referenceSequenceNumber < checkpoint.protocolState.sequenceNumber) {
             return {
                 message: {
                     errorMessage: `Proposed summary reference sequence number ${op.referenceSequenceNumber} is less than current sequence number ${op.sequenceNumber}`,
```

Comment:
This message does not seem to match the IF condition

Reply:
The statement checks if the clients summary op sequence number is before the current protocol state sequence number. If it is, then we prevent their summary

Reply:
sure, but could you please change message to 
> less than current sequence number ${protocolSequenceNumber}`
to match the IF condition.

Reply:
I updated it and updated the error message. The message was including the wrong sequence number, which made this extra confusing